packages/smart/smart_p.go

```diff
@@ -664,21 +666,23 @@ func RollbackEcosystem(sc *SmartContract) error {
 	return ssToDel.Delete(sc.DbTransaction)
 }
 
+// RollbackTable is rolling back table
 func RollbackTable(sc *SmartContract, name string) error {
 	if sc.TxContract.Name != `@1NewTable` {
 		log.WithFields(log.Fields{"type": consts.IncorrectCallingContract}).Error("RollbackTable can be only called from @1NewTable")
 		return fmt.Errorf(`RollbackTable can be only called from @1NewTable`)
 	}
-	err := model.DropTable(sc.DbTransaction, fmt.Sprintf("%d_%s", sc.TxSmart.EcosystemID, name))
+	model.DropTable(sc.DbTransaction, fmt.Sprintf("%d_%s", sc.TxSmart.EcosystemID, name))
```

Comment:
I think
if err != nil {
		log.WithFields(log.Fields{"type": consts.DBError, "error": err}).Error("deleting table")
		return err
	}
should be after DropTable

Reply:
i agree